packages/components/Menu/package.json

```diff
@@ -33,6 +33,7 @@
     "@fluentui-react-native/icon": "^0.17.24",
     "@fluentui-react-native/interactive-hooks": ">=0.22.28 <1.0.0",
     "@fluentui-react-native/text": ">=0.19.30 <1.0.0",
+    "@fluentui-react-native/divider": "^0.3.14",
```

Comment:
I don't see this used in your test changes, can we remove this addition? If it should be there it would make sense to have it be its own PR.

Reply:
Actually, It is used in packages/components/Menu/src/MenuDivider/MenuDivider.types.ts . 
But i think having the entire Divider dependency for just a type didn't make sense. 

So i have localised type to MenuDivider.

Reply:
Oh I misread which package.json that was. My bad.